<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>270</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Flash Programmer v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/perfix1/img/property_obj.gif</normaloff>:/new/perfix1/img/property_obj.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_comno">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>430</width>
      <height>10</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>430</width>
      <height>5</height>
     </size>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_com">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>26</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_baudrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>39</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_refresh">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_notice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>390</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;开始写入之前，请确认设备已通过串口连接，并且已处于接收状态。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_braudrate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_bigpic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/new/perfix1/img/news.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_helppic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/new/perfix1/img/help.gif</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_srcflie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>52</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>目标文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_srcfile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_choice">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_statuspic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_write">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>写入</string>
    </property>
   </widget>
   <zorder>comboBox_comno</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_com</zorder>
   <zorder>label_baudrate</zorder>
   <zorder>Button_close</zorder>
   <zorder>Button_refresh</zorder>
   <zorder>label_notice</zorder>
   <zorder>comboBox_braudrate</zorder>
   <zorder>label_bigpic</zorder>
   <zorder>label_helppic</zorder>
   <zorder>label_srcflie</zorder>
   <zorder>lineEdit_srcfile</zorder>
   <zorder>Button_choice</zorder>
   <zorder>Button_write</zorder>
   <zorder>label_status</zorder>
   <zorder>label_statuspic</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox_comno</tabstop>
  <tabstop>Button_refresh</tabstop>
  <tabstop>comboBox_braudrate</tabstop>
  <tabstop>lineEdit_srcfile</tabstop>
  <tabstop>Button_choice</tabstop>
  <tabstop>Button_write</tabstop>
  <tabstop>Button_close</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
